src/unix/core.c

```diff
@@ -1677,7 +1677,7 @@ int uv_thread_setpriority(uv_thread_t tid, int priority) {
     param.sched_priority = prio;
     r = pthread_setschedparam(tid, policy, &param);
     if (r != 0)
-      return UV__ERR(errno);  
+      return UV__ERR(errno);
```

Comment:
Nit: I would not include the unrelated style changes so git blame is more useful